entry/src/main/resources/base/element/string.json

```diff
@@ -0,0 +1,16 @@
+{
+  "string": [
+    {
+      "name": "entry_MainAbility",
+      "value": "entry_MainAbility"
```

Comment:
Please modify the default string values from generic values to appropriate values based on the library. If the strings are not required, you can delete them